client/src/components/CreateAccount.js

```diff
@@ -57,40 +57,40 @@ class CreateAccount extends Component {
                         <Form onSubmit={this.createAccount}>
                             <Form.Row>
                                 <Form.Group as={Col} controlId="formGridUsername">
-                                <Form.Label>Username</Form.Label>
-                                <Form.Control name="username" type="username" placeholder="Username" />
+                                    <Form.Label>Username</Form.Label>
+                                    <Form.Control name="username" type="username" placeholder="Username" />
                                 </Form.Group>
 
                                 <Form.Group as={Col} controlId="formGridEmail">
-                                <Form.Label>Email</Form.Label>
-                                <Form.Control name="email" type="email" placeholder="Email" />
+                                    <Form.Label>Email</Form.Label>
+                                    <Form.Control name="email" type="email" placeholder="Email" />
                                 </Form.Group>
                             </Form.Row>
 
                             <Form.Row>
                                 <Form.Group as={Col} controlId="formGridFirst">
-                                <Form.Label>First Name</Form.Label>
-                                <Form.Control name="first" type="first" placeholder="First name" />
+                                    <Form.Label>First Name</Form.Label>
+                                    <Form.Control name="first" type="first" placeholder="First name" />
                                 </Form.Group>
 
                                 <Form.Group as={Col} controlId="formGridLast">
-                                <Form.Label>Last Name</Form.Label>
-                                <Form.Control name="last" type="last" placeholder="Last name" />
+                                    <Form.Label>Last Name</Form.Label>
+                                    <Form.Control name="last" type="last" placeholder="Last name" />
                                 </Form.Group>
                             </Form.Row>
 
                             <Form.Row>
                                 <Form.Group as={Col} controlId="formGridPassword">
-                                <Form.Label>Password</Form.Label>
-                                <Form.Control name="password" type="password" placeholder="Password" />
+                                    <Form.Label>Password</Form.Label>
+                                    <Form.Control name="password" type="password" placeholder="Password" />
                                 </Form.Group>
                             </Form.Row>
 
                             <div style={{ "text-align": "right" }}>
                                 <Link to="/">
                                     <Button variant="primary" type="submit">
                                         Login
-                                    </Button>{' '}
```

Comment:
Don't delete this, it keeps a space between the buttons instead of merging them together